packages/example-rpc-server/README.md

```diff
@@ -1,14 +1,29 @@
-# loopback4-example-rpc-server
+# @loopback/example-rpc-server
 
 An example RPC server and application to demonstrate the creation of your
 own custom server.
 
 [![LoopBack](http://loopback.io/images/overview/powered-by-LB-xs.png)](http://loopback.io/)
 
 ## Usage
-Install dependencies and start the app:
-```sh
-npm install
+
+1. Install the new loopback CLI toolkit.
+```
+npm i -g @loopback/cli
+```
+
+2. Download the "getting-started" application.
```

Comment:
I'm guessing you're adding this to the `lb4 example` prompt and editing these instructions?